src/content/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-migration-guide.mdx

```diff
@@ -0,0 +1,316 @@
+        </td>
+        <td>
+          Since each database is reported as a dimension, there's no direct replacement for `totalDatabases`.
+
+          You can obtain the total number of databases by counting the unique values under the `database` dimension:
+
+          ```sql
+          FROM Metric SELECT uniqueCount(database) WHERE mongodb_cluster_name='your_cluster'
+          ```
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+  </Collapser>
+
+  <Collapser
+    id="top-samples"
+    title="Top Samples"
+  >
+    Metrics from the legacy `MongodTopSample` are reported under the prefix `mongodb_top_`.
+
+    You can use the `database` and `collection` dimensions to narrow down your queries to a specific database and/or collection.
+
+    <table>
+      <thead>
+      <tr>
+        <th>
+          Legacy metric
+        </th>
+        <th>
+          New metric
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          `usage.queriesPerSecond`
+        </td>
+        <td>
+          `mongodb_top_queries_count`
+        </td>
+      </tr>
+      <tr>
+        <td>
+          `usage.queriesInMillisecondsPerSecond`
+        </td>
+        <td>
+          `mongodb_top_queries_time`
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+    This is an example of query to get a time series of the frequency of insert commands:
+
+    ```sql 
+    SELECT derivative(mongodb_top_insert_time, 1 minute) / 1000 AS 'Inserts' Metric WHERE mongodb_cluster_name='your_cluster' TIMESERIES
+    ```
+
+  </Collapser>
+
+  <Collapser
+    id="mongo-samples"
+    title="Mongo Samples"
+  >
+    Mongo Sample reports the `totalDatabases` metric, which hasn't direct correspondence on the new integration.
+
+    To retrieve the total number of databases, see this example:
+
+    <table>
+      <thead>
+      <tr>
+        <th>
+          Legacy metric
+        </th>
+        <th>
+          New metric
+        </th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr>
+        <td>
+          `totalDatabases`
+        </td>
+        <td>
+          Since each database is reported as a dimension there isn't direct replacement for `totalDatabases`.
+          
+          You can obtain the total number of databases by counting the unique values under the `database` dimension:
+
+          ```sql
+          FROM Metric SELECT uniqueCount(database) WHERE mongodb_cluster_name='your_cluster'
+          ```
+        </td>
+      </tr>
+      </tbody>
+    </table>
+
+  </Collapser>
+
+  <Collapser
+    id="mongod-samples"
+    title="Mongod Samples"
+  >
+    Mongod metrics are the same as Config Server metrics but the `cl_role` dimension is populated with a value of `shardsvr`.
+
+  </Collapser>
+
+  <Collapser
+    id="mongos-samples"
+    title="Mongos Samples"
+  >
+    Mongos metrics are the same as Config Server metrics but the `cl_role` dimension is populated with a value of `mongos`.
+
```

Comment:
`Complete these steps on the mongos to be monitored. If mongod-level metrics are to be collected, such as host or replica set statistics, create the role and user on each mongod as well.`

I find this sentence complex and difficult to follow. (the callout it is in src/content/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-monitoring-integration-new.mdx file, but I cannot leave comments since the file is too big)

Reply:
Kind of agree. Is this callout really needed?

IMO this sentence "Complete these steps on the mongos to be monitored. "is already understood because we already are in the Configuration section.

About the other sentence in the callout "If mongod-level metrics are to be collected, such as host or replica set statistics, create the role and user on each mongod as well.". What if we add a third step stating that is only needed if certain metrics are needed (not really sure what mongod-level metrics are referring to).

Reply:
Modifications made